src/bin/server.rs

```diff
@@ -1,3 +1,4 @@
+#![recursion_limit = "256"]
```

Comment:
because it won't compile otherwise. There seems to be a limit to how many endpoints we can have, and we have reached that limit. Increasing the recursion time extends the number of endpoints we can have.